backend/src/database/seeds/10-Colors_prod.ts

```diff
@@ -0,0 +1,21 @@
+import { Knex } from 'knex';
+
+export async function seed(knex: Knex): Promise<void> {
+  // Deletes ALL existing entries
+  await knex('colors').del();
```

Comment:
can we copy over the reset code from the other seed files? those reset the primary index too. this doesnt

Reply:
ok!